src/design-tokens/mixins.css

```diff
@@ -7,17 +7,17 @@
 }
 
 @define-mixin textLink {
-  @mixin eds-theme-typography-body-text-md;
   font-size: inherit;
+  font-weight: var(--eds-font-weight-medium);
```

Comment:
ensures links are font weight "medium" (500), instead of "light" (400) -- double-checked with Figma presets